kannada/Best-Songs/yav-mohana-murali

```diff
@@ -0,0 +1,32 @@
+Below is a kannada song, which is favourite of many people.
+Translation of this song can be read [here](http://kannadahaadulyrics.blogspot.com/2011/05/yava-mohana-murali-kareyitu.html)
+This song was taken in a film and the video can be viewed [here](https://www.youtube.com/watch?v=iJ5EjxeBO3U)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.